src-backend/api/tests/utils/decorators.py

```diff
@@ -0,0 +1,9 @@
+import api.startup
+
+# Use this decorator to add permissions to a new user
+# created in your setUp method for tests.
+def add_group_permissions(self):
+    def _run_once():
```

Comment:
We have been using __ for private functions. Not sure if that'll cause issues here.